CHANGELOG.md

```diff
@@ -18,6 +20,214 @@
+    a body [#4039](https://github.com/Kong/kong/pull/4039)
+- New config options:
+  - `dns_valid_ttl` [#3730](https://github.com/Kong/kong/pull/3730)
+  - `pg_timeout` [#3808](https://github.com/Kong/kong/pull/3808)
+  - `upstream_keepalive` can be disabled when set to 0 (thanks,
+    [@pryorda](https://github.com/pryorda)!) [#3716](https://github.com/Kong/kong/pull/3716)
+  - The new `transparent` suffix also applies to the `proxy_listen` directive
+- Cluster-wide mutex [#3685](https://github.com/Kong/kong/pull/3685)
+- Add multipart support to Admin API: [#3776](https://github.com/Kong/kong/pull/3776)
+- Improved plugins iterator speed [#3794](https://github.com/Kong/kong/pull/3794)
+- Add support for HTTPS in active healthchecks [#3815](https://github.com/Kong/kong/pull/3815)
+
+##### Plugins
+
+- New options `no_api`, `no_route` & `no_service` & `no_consumer` to limit
+  where plugins can be attached to [#3646](https://github.com/Kong/kong/pull/3646)
+- The `http-log` plugin now accepts buffered logging [#3604](https://github.com/Kong/kong/pull/3604)
+- Most plugin logic has been rewritten with the PDK instead of using internal
+  kong functions or ngx calls [#3845](https://github.com/Kong/kong/pull/3845)
+- Plugins in general have been made more resilient to failures / unexpected input
+  [#4006](https://github.com/Kong/kong/pull/4006),
+  [#3947](https://github.com/Kong/kong/pull/3947),
+  [#4038](https://github.com/Kong/kong/pull/4038)
+- AWS now supports Lambda Proxy Integration
+
+### Fixes
+
+##### Core
+
+- Nested records are validated by the metaschema [#3588](https://github.com/Kong/kong/pull/3588)
+- `kong.db.errors` receives the name of the current strategy [#3612](https://github.com/Kong/kong/pull/3612)
+- The new DAO has the correct CRUD event [#3659](https://github.com/Kong/kong/pull/3659)
+  and triggers update events only once [#4095](https://github.com/Kong/kong/pull/4095)
+- SNIs are correctly paginated [#3722](https://github.com/Kong/kong/pull/3722)
+- null & default values are handled better
+  [#3772](https://github.com/Kong/kong/pull/3772),
+  [#3710](https://github.com/Kong/kong/pull/3710),
+  [#3910](https://github.com/Kong/kong/pull/3910)
+- Inferring arguments with `application/x-www-form-urlencoded` on the admin API
+  works better [#3770](https://github.com/Kong/kong/pull/3770)
+- The cache_key field is properly calculated [#3831](https://github.com/Kong/kong/pulls/3831)
+- `$request_uri` being nil now is handled correctly [#3842](https://github.com/Kong/kong/pull/3842)
+- Primary key violations in Cassandra are detected more accurately [#3865](https://github.com/Kong/kong/pull/3865)
+- Datastax Enterprise 6.X doesn't throw errors (thanks, [@gchristidis](https://github.com/gchristidis)) [#3873](https://github.com/Kong/kong/pull/3873)
+- Posgresql's `connector.iterate` function is used properly [#3887](https://github.com/Kong/kong/pull/3887)
```

Comment:
`Postgres'` or `PostgreSQL's`

Reply:
I ended up removing that line completely